main/src/ca/uwaterloo/flix/language/errors/LexerError.scala

```diff
@@ -196,5 +197,49 @@ object LexerError {
 
     override def explain(formatter: Formatter): Option[String] = None
   }
-}
 
+  /**
+   * An error raised when an unterminated string is encountered.
+   *
+   * @param loc The location of the opening `{`.
+   */
+  case class UnterminatedStringInterpolation(loc: SourceLocation) extends LexerError {
+    override def summary: String = s"Unterminated string interpolation."
+
+    override def message(formatter: Formatter): String = {
+      import formatter._
+      s"""${line(kind, source.name)}
+         |>> Unterminated string interpolation.
```

Comment:
Something about missing end brance?

Reply:
I think all the `Unterminated*` errors would be better off with a `Missing '_' in ____`, so they are all similar.
I already made that change.